specification/configuration.md

```diff
@@ -92,6 +92,7 @@ environment variables:
 | :------------------------------------:          | :--------------------------------------------------------:     |
 | `SPLUNK_ACCESS_TOKEN` ()                        | Access token added to exported data. [1]                       |
 | `SPLUNK_TRACE_RESPONSE_HEADER_ENABLED` (`true`) | Whether `Server-Timing` header is added to HTTP responses. [2] |
+| `SPLUNK_METRICS_ENDPOINT` ()                    | Endpoint for non-OTel metrics data ingest.                     |
```

Comment:
```suggestion
| `SPLUNK_METRICS_ENDPOINT` ()                    | Endpoint for metrics data ingest.                     |
```

Reply:
Just to confirm @mateuszrzeszutek this is what splunk-otel-java uses?

Reply:
In the name of Mateusz - yes it is.